src/realm/sync/client.cpp

```diff
@@ -1102,6 +1106,28 @@ bool SessionImpl::is_steady_state_download_message(DownloadBatchState batch_stat
     return false;
 }
 
+void SessionImpl::update_download_estimate(double download_estimate)
+{
+    if (m_state == State::Active) {
+        m_wrapper.m_download_estimate = download_estimate;
+    }
+}
+
+void SessionImpl::notify_download_progress(const std::optional<uint64_t>& transient_changesets_size)
+{
+    if (m_state == State::Active) {
+        if (transient_changesets_size) {
```

Comment:
After reading through the PR I still don't quite get what the difference between the transient changeset size and the downloaded bytes info we store in history is.

Reply:
i've simplified a bit the code around it so that it's clear where it is used - essentially these are bootstrap store pending bytes, it's not used anywhere else